plugins/woocommerce/src/Blocks/BlockTypes/ProductButton.php

```diff
@@ -218,7 +218,7 @@ protected function render( $attributes, $content, $block ) {
 		';
 
 		$button_directives = $is_descendant_of_add_to_cart_form ?
-			'data-wp-class--disabled="woocommerce/add-to-cart-with-options::!state.isFormValid"' :
+			'data-wp-bind--disabled="woocommerce/add-to-cart-with-options::!state.isFormValid"' :
```

Comment:
I would leave this as-is, we want the button to look disabled but still be clickable as this is the recommended accessibility approach nowadays. This way, shoppers can still click on it and see the error of _why_ it is disabled.